File: estudos-alp-nelio-alves-main/Java/Vetores.java
import java.util.Locale;
import java.util.Scanner;

public class Vetores {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner scanner = new Scanner(System.in);

        System.out.print("Quantos números você vai digitar? ");
        int n = scanner.nextInt();

        double[] vet = new double[n];

        for (int i = 0; i < n; i++) {
            System.out.print("Digite um número: ");
            vet[i] = scanner.nextDouble();
        }

        System.out.println("\nNúmeros digitados:");
        for (int i = 0; i < n; i++) {
            System.out.printf("%.1f%n", vet[i]);
        }

        scanner.close();
    }
}

File: estudos-alp-nelio-alves-main/c#/vetores.cs
using System;
using System.Globalization;

namespace Vetores
{
    class Program
    {
        static void Main(string[] args)
        {
            CultureInfo CI = CultureInfo.InvariantCulture; // Define cultura invariante para formatação
            int N;
            Console.Write("Quantos números você vai digitar? ");
            N = int.Parse(Console.ReadLine());

            double[] vet = new double[N];
            for (int i = 0; i < N; i++)
            {
                Console.Write("Digite um número: ");
                vet[i] = double.Parse(Console.ReadLine(), CI);
            }
            Console.WriteLine();
            Console.WriteLine("Números digitados:");
            for (int i = 0; i < N; i++)
            {
                Console.WriteLine(vet[i].ToString("F2", CI));
            }
      

        }

    }
}
